### PR TITLE
style(packages/app): lint unused-vars

### PR DESCRIPTION
**Pull request by jesusprubio**
#### Please confirm that your PR fulfills these requirements
- [x] Multiple commits are squashed into one commit.
- [x] The commit message follows [Conventional Commits](https://github.com/IBM/kui/blob/master/CONTRIBUTING.md#conventional-commits), which allows us to autogenerate release notes; e.g. `fix(plugins/plugin-k8s): fixed annoying bugs`
- [x] All npm dependencies are pinned.

### CODE REVIEW

File: packages/app/src/core/plugin-assembler.ts [hunk 1]
```diff
@@ -18,19 +18,17 @@ import * as Debug from 'debug'
...
 const debug = Debug('core/plugin-assembler')
-debug('loading')
-import mkdirp = require('mkdirp')
+// debug('loading')
 
-const TMP = 'plugins' // we'll stash the original plugins here
-const TMA = 'app' // we'll stash the original app here
+// // TODO
+// const TMP = 'plugins' // we'll stash the original plugins here
+// const TMA = 'app' // we'll stash the original app here
```
Comment (myan9) [inline packages/app/src/core/plugin-assembler.ts]: Thanks! TMP and TMA are outdated indeed. It's ok to delete them! 😄

Comment (jesusprubio) [reply packages/app/src/core/plugin-assembler.ts]: The result after dropping those lines was a bit strange:

image

So I changed it to this. Is what we want?

image

File: packages/app/src/core/userdata.ts [hunk 2]
```diff
@@ -115,21 +115,6 @@ const fsyncPreferences = (prefs: Preferences): Preferences => {
   return prefs
 }
 
-/**
```
Comment (starpit) [inline packages/app/src/core/userdata.ts]: let's keep this, and `export` it

File: packages/app/src/core/plugin-assembler.ts (hunk 1 above)
Comment (myan9) [reply packages/app/src/core/plugin-assembler.ts]: yes seems good to me, then we can know the module loading time. thanks!

**Comment by jesusprubio** [comment]
I've just rebased, anything else? :smile:

**Pull request merged**